android/src/main/java/com/vydia/RNUploader/Upload.kt

```diff
@@ -72,8 +72,8 @@ class Upload(options: ReadableMap) {
     if (options.hasKey("maxRetries") && options.getType("maxRetries") == ReadableType.Number)
       maxRetries = options.getInt("maxRetries")
 
-    if (options.hasKey("isDiscretionary"))
-      discretionary = options.getBoolean("isDiscretionary")
```

Comment:
should rename other "discretionary" variables to "wifiOnly" as well.

Reply:
Yeah, I thought about doing that. But there are so many variables in this file that have that name, and it's not really confusing in the Android context since it's not an android API name. I don't know whether it's a good idea

Reply:
wait, all the `isDiscretionary` were your changes in our fork?